packages/cli/src/core/config-engine.ts

```
@@ -243,7 +243,8 @@ export class ConfigEngine {
     }
 
     const timestamp = new Date().toISOString().replace(/[:.]/g, '-');
-    const backupFileName = `${basename(configPath)}.${timestamp}.backup`;
+    const sanitizedConfigPath = configPath.replace(/[:\/\\]/g, '_');
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)